config/registry/aws/modules/cloudfront-distribution.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "cloudfront-distribution"
+linkTitle: "cloudfront-distribution"
+date: 2021-11-16
+draft: false
+weight: 1
+description: Set up a cloudfront distribution to serve static files from your s3 bucket
```

Comment:
This should just say "Set up a cloudfront distribution".

Reply:
done